docs/react/week-2/homework.md

```diff
@@ -6,46 +6,28 @@ sidebar_label: Coursework
+Complete all of the Lesson 2 exercises in the `cyf-hotel-react` project: https://github.com/CodeYourFuture/cyf-hotel-react#lesson-2.
 
-https://github.com/CodeYourFuture/cyf-hotel-react#lesson-2
+## 3) Challenges (10 hours+) (Optional, but recommended)
 
-## 3) High Score Table Project - Part 2 (7 Hours)
+The **best** way to get better at React is to practice by building small projects.
 
-The key way to get better at React is to build lots of small projects and build up your knowledge.
+Continue to work on the challenge projects from the previous week. You should now be able to complete more levels of the challenge
 
-Suitable projects for you to work on right now are:
+We recommend that you complete the following challenge:
 
-- High Score Table - All Levels
-  - https://github.com/CodeYourFuture/cyf-react-challenges/tree/master/challenge-high-score-tables) - All levels
+- [High Score Table - all remaining levels](https://github.com/CodeYourFuture/cyf-react-challenges/tree/master/challenge-high-score-tables)
 
-You should also choose any of the other projects that sound interesting to you!
+Some other suitable projects for you to work on right now are:
 
-## 4) (Optional) More Challenges (10 hours+)
+- [Baby Names Picker - Level 2](https://github.com/CodeYourFuture/cyf-react-challenges/tree/master/challenge-baby-name-picker)
+- [Countries - Level 2](https://github.com/CodeYourFuture/cyf-react-challenges/tree/master/challenge-countries)
+- [Job Listing - Level 2](https://github.com/CodeYourFuture/cyf-react-challenges/tree/master/challenge-job-listing)
 
-You can see a full list of the challenges here:
+## 4) Prepare for the next class
 
-https://github.com/CodeYourFuture/cyf-react-challenges/
-
-Suitable projects for you to work on right now are:
-
-- Baby Names Picker - Level 2
-  - https://github.com/CodeYourFuture/cyf-react-challenges/tree/master/challenge-baby-name-picker
-- Countries - Level 2
-  - https://github.com/CodeYourFuture/cyf-react-challenges/tree/master/challenge-countries
-- Job Listing - Level 2
-  - https://github.com/CodeYourFuture/cyf-react-challenges/tree/master/challenge-job-listing
-
-## 5) Prepare for the next class
-
-1. Finish reading the official [React Tutorial](https://reactjs.org/tutorial/tutorial.html)
-   - You can ignore anything that uses `Class Components`. This is any code that includes `extends React.Component`
-   - This is because we use a slightly newer version of React that uses `Functional Components`
-   - You can read [this overview](https://dev.to/danielleye/react-class-component-vs-function-component-with-hooks-13dg) for more details on the differences
-2. Read the [lifecycle page](https://reactjs.org/docs/state-and-lifecycle.html) on the official React documentation
-3. Read the [AJAX page](https://reactjs.org/docs/faq-ajax.html) on the official React documentation
-4. Watch [this video tutorial](https://www.youtube.com/watch?v=TxqqrNfgTto) on fetching data in React
+TODO
```

Comment:
When do we expect to resolve this TODO? In this PR? In the coming weeks?

Reply:
II think this TODO can be safely removed, these changes merged, and a new pullreq opened when it has taken shape.

Reply:
Yeah, I thought someone might raise this 😄 

As noted in the review notes, all the links here was outdated so I don't think we should include it. I think I did mean to put a call out on the React module channel for good, up-to-date content but I apparently forgot.

I'll go ahead and remove this, since I don't think it's helpful for trainees to see this.

Reply:
Done